Synview-backend/routes/github_routes.ts

```diff
@@ -70,6 +72,48 @@ githubRouter
       };
     }
   })
-  .get("/getCommit", async (context) => {});
+  .get("/getCommitFiles", async (context) => {
+    try {
+      const url = context.request.url;
+      const github_user = url.searchParams.get("user");
+      const repo_name = url.searchParams.get("repo");
+      const commit_sha = url.searchParams.get("sha");
+      if (!github_user || !repo_name || !commit_sha) {
+        throw new Error("Didnt find url info");
```

Comment:
Using the logger (maybe the code we talked about last week) I'd recommend logging additional error info here so that at least in the debug logs, you can see exactly what happened (ie. was the `repo_name` null but the other params were nonnull? etc